nbviewer/templates/index.html

```diff
@@ -1,10 +1,15 @@
 {% extends "layout.html" %}
 
 {% block body %}
-  {% if not no_frontpage_input %}
```

Comment:
The origin `no_frontpage_input` flag also hid the jumbotron header. Does the page look OK with that element still in the DOM when there's no title, subtitle, or text?

Reply:
Let me check on that, and get back to you.

Reply:
I've added a new commit to hide the jumbotron if nothing is contained within it.